### PR TITLE
TT-173 - Change method for reading pending transactions to sign

### PR DESCRIPTION
**Pull request by isra67**

### CODE REVIEW

File: src/signatures.ts [hunk 1]
```diff
@@ -389,6 +388,40 @@ const processTransaction = async (
...
+  }
+
+  console.log(
+    `${new Date().toISOString()} - Getting pending transaction from ${chain} for ${
+      signatureIds.length > OUTPUT_LIMIT ? signatureIds.length + ' ' : ''
+    }wallets${signatureIds.length > OUTPUT_LIMIT ? '' : ' ' + signatureIds.join(',')}.`,
+  )
+  try {
+    const url = (process.env.TATUM_API_URL || 'https://api-eu1.tatum.io') + `/v3/kms/pending/${chain}`
+    const { data } = await axios.post(
```
Comment (npwork) [inline src/signatures.ts:410]: Maybe you can use @tatumio/api-client for that?

File: src/signatures.ts [hunk 2]
```diff
@@ -389,6 +388,40 @@ const processTransaction = async (
   })
 }
 
+const getPendingTransactions = async (
+  axios: AxiosInstance,
+  chain: Currency,
+  signatureIds: string[],
+): Promise<TransactionKMS[]> => {
+  const LIMIT = 25_000
+  const OUTPUT_LIMIT = 5
```
Comment (npwork) [inline src/signatures.ts:397]: extract to constants file

File: src/signatures.ts (hunk 1 above)
Comment (isra67) [reply src/signatures.ts:411]: I think apply new SDK will occur in next round

**Comment by samuelsramko** [comment]
@isra67 WE are missing README information what is happening and limitations which are being introduced with this change.

**Review by samuelsramko — APPROVED** [review]

**Pull request merged**